### PR TITLE
Event api

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: events/event/views.py [hunk 1]
```diff
@@ -1,3 +1,79 @@
...
+from .models import Event, EventUserAssignment, User
+from .forms import EventCreateForm
+
+# Variabels for converting string to datetime
+tz = get_current_timezone()
+format = '%b %d %Y %I:%M%p'
+
+class EventView(View):
+    def get(self, request, pk=None):
+        if not pk:
```
Comment (yevgenbehen) [inline events/event/views.py]: Get should return something ONLY when it gets identifier (object ID). It should never return everything from the database table.

File: events/event/views.py [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+from django.http import JsonResponse, HttpResponse
+from django.views.generic.base import View
+from django.core import serializers
+
+from .models import Event, EventUserAssignment, User
+from .forms import EventCreateForm
+
+# Variabels for converting string to datetime
+tz = get_current_timezone()
+format = '%b %d %Y %I:%M%p'
```
Comment (yevgenbehen) [inline events/event/views.py]: This should be constant - with all capital letters

File: events/event/views.py [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+            event = Event.objects.get(id=pk)
+        except:
+            return HttpResponseNotFound('Does not exist')
+        body_unicode = request.body.decode('utf-8')
+        data = json.loads(body_unicode)
+        data["start_date"] = tz.localize(datetime.datetime.strptime(data["start_date"], format))
+        data["end_date"] = tz.localize(datetime.datetime.strptime(data["end_date"], format))
+        form = EventCreateForm(data)
+        if form.is_valid():
+            for k, v in dates.items():
```
Comment (yevgenbehen) [inline events/event/views.py]: Where do you create `dates` variable?

**Review by yevgenbehen — CHANGES_REQUESTED** [review]

File: events/event/views.py (hunk 3 above)
Comment (ghost) [reply events/event/views.py]: Line #37

**Review by yevgenbehen — APPROVED** [review]

**Review by yevgenbehen — APPROVED** [review]

**Comment by yevgenbehen** [comment]
+1 from me

**Review by lhalam — APPROVED** [review]

**Pull request merged**